sql/moz-fx-data-shared-prod/addons/addon_events_clients/view.sql

```diff
@@ -0,0 +1,37 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.addons.addon_events_clients`
+AS
+SELECT
+  submission_date,
+  `timestamp`,
+  COALESCE(
+    `moz-fx-data-shared-prod.udf.get_key`(event_map_values, 'addonId'),
+    `moz-fx-data-shared-prod.udf.get_key`(event_map_values, 'addon_id')
+  ) AS addon_id,
+  COALESCE(
+    `moz-fx-data-shared-prod.udf.get_key`(event_map_values, 'addonVersion'),
+    `moz-fx-data-shared-prod.udf.get_key`(event_map_values, 'addon_version')
+  ) AS addon_version,
+  -- TODO: why in some redash queries this is hashed_addon_id and in other value? What's the difference?
+  event_string_value AS hashed_addon_id,
+  client_id,
+  sample_id,
+  normalized_channel,
+  country,
+  locale,
+  app_name,
+  app_version,
+  os AS os_name,
+  os_version,
+  event_timestamp,
+  event_category,
+  event_method AS method,
+  event_object AS `object`,
+  event_string_value AS `value`,
+  event_map_values,
+  event_process,
+FROM
+  `moz-fx-data-shared-prod.telemetry.events`
+WHERE
+  submission_date >= '2021-08-09'
+  AND event_category IN ("addonsManager", "addonsSearchDetection")
```

Comment:
I think it'd be good to _not_ do this. `addonsSearchDetection` is its very own thing, could we keep it self-contained by chance?

Reply:
this is just a view on top of telemetry.events just to make it a bit easier to query the addons data and prevent other event categories to be included in the results in case the filter is not included in your query. event_category field is still accessible in this view and can still be used to filter for a specific group of events in the follow up queries. Like we do in search_detection_v1 for example:
https://github.com/mozilla/bigquery-etl/pull/6827/files/774329c67a7bc861d6945d1aadeb70e13f0456b7#diff-a9127827452486e083c54ec89dee402563de06263285a5ef981e53cf50215503R22

Reply:
Yes, based on the other comment you left along with the description and explanation that hashed_addon_id is only in some cases I removed the alias.